mgmt_cli_test.py

```diff
@@ -831,12 +830,11 @@ def test_enospc_before_restore(self):
                     f"The restore task is supposed to fail, since node {target_node} lacks the disk space to download" \
                     f"the snapshot files"
 
-                if not SkipPerIssues("scylladb/scylla-manager#4087", self.params):
-                    full_progress_string = restore_task.progress_string(parse_table_res=False,
-                                                                        is_verify_errorless_result=True).stdout
-                    assert "not enough disk space" in full_progress_string.lower(), \
-                        f"The restore failed as expected when one of the nodes was out of disk space, " \
-                        f"but with an ill fitting error message: {full_progress_string}"
+                full_progress_string = restore_task.progress_string(parse_table_res=False,
+                                                                    is_verify_errorless_result=True).stdout
+                assert "failed to restore sstables from location" in full_progress_string.lower(), \
```

Comment:
For now, I'd probably drop this assertion. 
In case of current, very common error message, it actually doesn't make much sense.
In future when we implement this, let's say, "smart" free space checker you mentioned, we can think about this test rework.